GitUI/GitUICommands.cs

```diff
@@ -1067,6 +1067,7 @@ public bool StartCherryPickDialog(IWin32Window owner, GitRevision revision)
             return DoActionOnRepo(owner, true, true, PreCherryPick, PostCherryPick, action);
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2202:Do not dispose objects multiple times")]
```

Comment:
:bulb: in hope that there is a better approach, I would prefer to see this moved to its own (separate) pull request.